includes/components/class-admin.php

```diff
@@ -244,6 +244,12 @@ public function init_settings() {
 				}
 			}
 		}
+
+		if ( 'hivepress/v1/activate' === current_filter() ) {
+
+			// Delete cached extensions.
+			hivepress()->cache->delete_cache( 'all_extensions' );
```

Comment:
It's ok if you create a new callback for this because it doesn't match this function name anyway, optimizing hook/callback pairs makes sense but to some extent

If this was indeed a caching issue, maybe let's also clear the cache for themes?